frontend/src/components/pages/AdminPage.tsx

```diff
@@ -1,35 +1,112 @@
 import React from "react";
-import { useHistory } from "react-router-dom";
-import * as Routes from "../../constants/Routes";
+import {
+  Flex,
+  Text,
+  Box,
+  Tabs,
+  TabList,
+  Tab,
+  TabPanel,
+  TabPanels,
+} from "@chakra-ui/react";
+import SideBar from "../common/Sidebar";
 
-import Logout from "../auth/Logout";
-import RefreshCredentials from "../auth/RefreshCredentials";
-import ResetPassword from "../auth/ResetPassword";
+import Page from "../../types/PageTypes";
+import AdminUserTable from "../common/AdminUserTable";
+import AddAdminModal from "../common/AddAdminModal";
 
-type ButtonProps = { text: string; path: string };
+const pages: Page[] = [
+  { title: "Assessments", url: "/" },
+  { title: "Database", url: "/" },
+];
```

Comment:
let's add the correct icons here! you may need to pull from master, but Wais just merged a PR that allows you to pass icons into the sidebar and design has designated specific icons for this design

Reply:
done :)